pre-commit

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-source dev-tools/check-code-style/check_style.sh
+source submodules/dev-tools/check-code-style/check_style.sh
```

Comment:
But this only works if someone includes `dev-tools` in `submodules`, what if they put it in `my-favorite-directory`?

Reply:
Is it used anywhere else except `eventuals`? Seems if we use `dev-tools` that means that we add it as a submodule and if it is, a user can't just bootstrap and use, because it would cause errors, isn't it?